### PR TITLE
Switch to new wrappers

### PR DESCRIPTION
**Pull request by Anze1m**

### CODE REVIEW

File: src/tasks/dev_deploy.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const logger = new WinstonConsoleLogger();
 
 createWeb3(config)
   .then(async (web3) => {
-    await deployAll(web3, logger);
+    await deployAll(web3, config.nodePrivateKey, logger);
```
Comment (spherefoundry) [inline src/tasks/dev_deploy.js]: we could use the new devConfig.deployerPrivateKey

File: test/helpers/server_apparatus.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ export default class ServerApparatus extends Builder {
     const web3 = _web3 || await createWeb3(this.config);
 
     if (!config.headContractAddress) {
-      const headContract = await deployAll(web3, this.logger);
+      const headContract = await deployAll(web3, config.nodePrivateKey, this.logger);
```
Comment (spherefoundry) [inline test/helpers/server_apparatus.js]: Same here `devConfig`

**Comment by danihodovic** [comment]
Works on staging.

File: src/tasks/dev_prerun_atlas.js [hunk 3]
```diff
@@ -10,21 +10,22 @@ This Source Code Form is “Incompatible With Secondary Licenses”, as defined
...
 import config from '../../config/config';
+import devConfig from '../../config/devConfig';
 import {addToKycWhitelist, onboardAsAtlas, registerAdminAccount} from '../utils/prerun';
 import {Role} from '../services/roles_repository';
 
-async function setupDevelopment(dataModelEngine, logger) {
+async function setupDevelopment(web3, dataModelEngine, logger) {
   await registerAdminAccount(dataModelEngine, logger);
-  await addToKycWhitelist(Role.ATLAS, dataModelEngine, logger);
-  await onboardAsAtlas(dataModelEngine.contractManager.web3, dataModelEngine.rolesRepository, logger);
```
Comment (spherefoundry) [inline src/tasks/dev_prerun_atlas.js]: :+1:

**Pull request merged**